src/plone/restapi/pas/plugin.py

```diff
@@ -127,6 +146,51 @@ def authenticateCredentials(self, credentials):
 
         return (userid, userid)
 
+    @security.private
+    def updateCredentials(self, request, response, login, new_password):
```

Comment:
So, if I understand correctly, you're making this plugin act when logging in from Plone Classic, in addition to the other PAS plugins, right? Seems ok to me.

Reply:
Pretty much.  Only a little better because it obeys the PAS plugin interfaces configuration.  So site administrators can decide which plugins should "participate" on login/logout "events".